### PR TITLE
CNN index.md

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-580/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

File: chapter_convolutional-neural-networks/index.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+在前面的章节中，我们曾讨论过图像数据。对于图像数据，每个示例都由一个二维像素网格组成。无论我们处理的是黑白图像还是彩色图像，每个像素可能与一个或多个数值相关。到目前为止，我们处理二维结构的方式还十分有限：先将每个图像的空间结构展平成一维向量，再放入一个多层感知机中。由于这些网络相对于特征元素的顺序不变，所以无论我们保持像素在二维空间的顺序，或是在拟合多层感知机的参数之前对二维矩阵的列进行置换，我们都可以训练相似的神经网络。而最优情况是利用先验知识，即附近像素之间的相互关联性，从而建立图像数据的有效模型。
 
-本章介绍 * 卷积神经网络 * (CNN)，这是一个功能强大的神经网络系列，专为此目的而设计。基于 CNN 的架构现在在计算机视觉领域普遍存在，并且已经变得如此主导，今天几乎没有人会开发商业应用程序或参与与图像识别、对象检测或语义分割相关的竞争，而不采用这种方法。
+本章介绍卷积神经网络（convolutional neural network，简称CNN）是一类强大的神经网络，正是为处理图像数据而设计的。基于卷积神经网络结构的模型在计算机视觉领域中已经占主导地位，当今几乎所有有关图像识别、对象检测或语义分割相关的学术竞赛或者商业应用都以这种方法为基础。
 
-现代有线电视网络，因为它们被称为口语地归功于来自生物学、群体理论和健康的实验修补剂量的灵感。除了实现精确模型的样本效率之外，CNN 在计算上往往具有高效率，因为它们所需的参数比完全连接的架构少，而且卷积很容易在 GPU 内核之间并行化。因此，从业人员经常在可能的情况下使用 CNN，而且他们越来越成为可信的竞争对手，即使在具有一维序列结构的任务上，例如音频、文本和时间序列分析，这些任务常常使用循环神经网络。CNN 的一些巧妙调整也使它们对图形结构数据和推荐系统产生了影响。
+现代卷积神经网络的设计得益于生物学、群论和大量的实验研究。它能获得精确模型，在采样和计算上也极其高效的。这是因为卷积神经网络需要的参数比多层感知机少，而且卷积神经网络很容易在GPU内核之间并行运算。因此，实践者经常尽可能多地应用卷积神经网络。即使在那些适合应用循环神经网络的一维序列结构的任务上（例如音频、文本和时间序列分析），卷积神经网络的应用也非常流行。此外，经巧妙改造后的卷积神经网络也常用于图结构数据和推荐系统应用。
```
Comment (xiaotinghe) [inline chapter_convolutional-neural-networks/index.md:8]: - “在GPU内核之间并行运算”-》 “跨GPU核心并行计算”
- “即使在一维序列结构的任务上（例如音频、文本和时间序列分析），实践者也会经常使用卷积神经网络。在这类任务上，通常大家使用的是循环神经网络。而卷积神经网络越来越成为有效的竞争者”
通过对cnn进行一些巧妙的调整，也使得它们在图结构数据和推荐系统中发挥作用。

File: chapter_convolutional-neural-networks/index.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-首先，我们将介入构成所有卷积网络骨干的基本操作。其中包括卷积层本身、包括填充和步长在内的细节、用于聚合相邻空间区域信息的池合图层、在每个层使用多个信道以及对现代架构结构的认真讨论。我们将以 Lenet 的完整工作示例结束本章，这是在现代深度学习崛起之前成功部署的第一个卷积网络。在下一章中，我们将深入了解一些流行和相对较新的 CNN 架构的完整实现，这些架构的设计代表了现代从业人员常用的大部分技术。
+在本章的开端，我们将介绍构成所有卷积网络主干的基本元素。这包括卷积层本身、填充（padding）和步幅（stride）的细节、用于在相邻空间区域聚集信息的池化层（pooling）、每层中多通道（channel）的使用以及有关现代卷积网络架构的全面介绍。在本章的最后，我们将介绍一个完整的、可运行的LeNet模型：这是第一个卷积神经网络，早在现代深度学习兴起之前就已经得到成功应用。在下一章中，我们将深入研究一些流行的、相对较新并具有一定代表性的卷积网络架构。
```
Comment (xiaotinghe) [inline chapter_convolutional-neural-networks/index.md:10]: "在本章的开端"->"首先"
“、”-》“；”

**Comment by mli** [comment]
Job d2l-zh/PR-580/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

File: chapter_convolutional-neural-networks/index.md (hunk 1 above)
Comment (goldmermaid) [reply chapter_convolutional-neural-networks/index.md:8]: "跨GPU核心并行计算" or  "跨GPU内核并行计算"?

**Comment by mli** [comment]
Job d2l-zh/PR-580/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/8 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/9 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/10 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/11 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/12 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

File: chapter_convolutional-neural-networks/index.md (hunk 1 above)
Comment (xiaotinghe) [reply chapter_convolutional-neural-networks/index.md]: 核心

Comment (goldmermaid) [reply chapter_convolutional-neural-networks/index.md]: or just "用GPU并行计算"？

Comment (xiaotinghe) [reply chapter_convolutional-neural-networks/index.md]: 用GPU并行计算吧

**Comment by mli** [comment]
Job d2l-zh/PR-580/13 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Comment by mli** [comment]
Job d2l-zh/PR-580/14 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-580/

**Pull request merged**